app/src/main/res/values/strings.xml

```diff
@@ -1027,6 +1027,9 @@
     <string name="etp_panel_off">Protections are OFF for this site</string>
     <!-- Header for exceptions list for which sites enhanced tracking protection is always off -->
     <string name="enhanced_tracking_protection_exceptions">Enhanced Tracking Protection is off for these websites</string>
```

Comment:
Nit: All comments for content descriptions follow this convention: 
    `<!-- Content description (not visible, for screen readers etc.): Navigate back from ETP details -->`

Reply:
Thanks! 😊